src/t_zset.c

```diff
@@ -468,33 +468,46 @@ unsigned long zslDeleteRangeByRank(zskiplist *zsl, unsigned int start, unsigned
     return removed;
 }
 
+#define zset_is_first_entry_same_score(x, score) ((!x->backward) || (x->backward->score < score))
+#define zset_is_last_entry_same_score(x, score) ((!x->level[0].forward) || (x->level[0].forward->score > score))
```

Comment:
```suggestion
#define zsl_is_first_entry_same_score(x, score) ((!x->backward) || (x->backward->score < score))
#define zsl_is_last_entry_same_score(x, score) ((!x->level[0].forward) || (x->level[0].forward->score > score))
```
Would it be clearer?